Source/DafnyCore/DafnyOptions.cs

```diff
@@ -313,7 +313,7 @@ public enum IncludesModes {
 
     public IncludesModes PrintIncludesMode = IncludesModes.None;
     public int OptimizeResolution = 2;
-    public bool UseRuntimeLib = false;
+    public bool IncludeRuntimeSource = true;
```

Comment:
I do think "include runtime" is better and less ambiguous than "use runtime lib" (since the latter is talking about what the user has to do and not what our code is doing, but I don't think adding source helps. The important meaning is to embed the runtime somehow, and It's conceivable some future backend might want to achieve this by statically linking a pre-built binary or something like that.

I would make this `IncludeRuntime` with a comment about what that actually means.